src/drivers/drv_sensor.h

```diff
@@ -113,6 +113,8 @@
 #define DRV_ACC_DEVTYPE_ADIS16497	0x63
 #define DRV_GYR_DEVTYPE_ADIS16497	0x64
 #define DRV_BARO_DEVTYPE_BAROSIM	0x65
+#define DRV_ACC_DEVTYPE_BMI088		0x66
```

Comment:
We actually only need a single devtype per physical device (ie BMI088). One per sensor type actually created a bit of a mess.

Reply:
It gets a bit complicated with the BMIs, though because they have two separate modules within the physical chip (accel and gyro), with two different chip-selects (and even two data output pins (although, those are usually coupled together in layout onto the MISO line))